web-frontend/cypress/support/e2e.ts

```diff
@@ -42,3 +42,40 @@ Cypress.on('window:before:load', (win) => {
     });
   };
 });
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import 'cypress-wait-until';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+Cypress.Commands.add('waitForRequest', (server, urlPattern, method = '') => {
+  const wildcardRegex = /\*/g;
+  const questionMarkRegex = /\?/g;
+
+  const replacedWildCard = urlPattern
+    .replace(wildcardRegex, '[^ ]*')
+    .replace(questionMarkRegex, '\\?');
+  const regexUrl = new RegExp(replacedWildCard, 'g');
```

Comment:
Let's refer here what that means:
Please, add a tiny comment (we are intercepting miragejs calls or related) and the link to the page:

https://github.com/miragejs/miragejs/issues/999#issuecomment-1495410040